kairon/chat/handlers/channels/voice.py

```
@@ -90,40 +98,91 @@ async def handle_incoming_call(self) -> Text:
+            agent = AgentProcessor.get_agent(self.bot)
+            tracker = await agent.tracker_store.retrieve(sender_id)
+            if not tracker:
+                return 0
+            count = 0
+            for event in reversed(tracker.events):
+                if isinstance(event, UserUttered) and event.text == "noinput":
+                    count += 1
+                elif isinstance(event, UserUttered):
+                    break
+            return count
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)